hooks/environment

```diff
@@ -26,15 +26,11 @@ processEnvSecrets() {
   local vaultServer="$1"
   local envKey="$2"
   local envscript=''
-
-  echo "Checking vault key ${envKey}" >&2
-  for key in $(list_secrets "$vaultServer" "$envKey")  ; do
-    echo "Downloading env secret from ${key}" >&2;
+  echo "Downloading env secret from ${key}" >&2;
     if ! envscript=$(echo "${envscript:-}" && secret_download "${vaultServer}" "${key}") ; then
       echo "+++ :warning: Failed to download env from $key" >&2;
```

Comment:
you removed the `for` stanza that was the one that defined the `key` variable here... aso you may have to re-indent the whole section

Reply:
Done, we now pass `$key` into the function correctly as well.